Index: init/init.cpp
IDEA additional info:
Subsystem: com.intellij.openapi.diff.impl.patch.CharsetEP
<+>UTF-8
===================================================================
--- init/init.cpp	(revision d3689d8a329fe472d782da94ec9166a42c65d8d8)
+++ init/init.cpp	(date 1526534099000)
@@ -431,7 +431,7 @@
 
     if (ret == -1) {
         LOG(ERROR) << "Unable to set adequate mmap entropy value!";
-        security_failure();
+        /* security_failure(); */
     }
     return ret;
 }
